update-format-crau/src/delta_update.rs

```diff
@@ -176,9 +184,17 @@ pub fn verify_sig_pubkey(digest: &[u8], sig: &Signature, pubkeyfile: &str) -> Op
     // in case of production update payloads. However, we do not explicitly check
     // for a signature version, as the number could differ in some cases.
     debug!("supported signature version: {:?}", sig.version());
+    if sig.version() == 1 {
```

Comment:
No, this is wrong, please see how `decode_payload` handles this. The version has no meaning, it's the index of the signatures and we should iterate over them until a valid one is found. If no valid one is found, we reject as unverified.

Reply:
Done